sites/partners/cypress/integration/listing.spec.ts

```diff
@@ -94,6 +94,13 @@ describe("Listing Management Tests", () => {
       cy.getByID("leasingAgentAddress.zipCode").type(listing["leasingAgentAddress.zipCode"])
       cy.getByID("leasingAgentAddress.state").select(listing["leasingAgentAddress.state"])
 
+      cy.getByTestId("mailing-address-checkbox").check()
```

Comment:
We have a bug right now on dev where you can't save a mailing address, fixed and added to Cypress